README.md

```diff
@@ -12,31 +12,88 @@ podman build -t eib:dev .
 
 ## Running
 
-At runtime, a volume must be mounted to the container. This serves as both the mechanism to introduce image
-configuration files and provide a way to get the built image out of the container and onto the host machine.
-
 **NOTE:** These docs are incomplete and will be fleshed out as the project matures. Below is an example that
 is being used for dev purposes. At some point when it's more mature, an example configuration directory will be
 added to this repository.
 
-Example image configuration directory:
+### Image Definition
+
+Two things are necessary to build an image using EIB:
+1. A configuration file that describes the image to build
+1. A directory that contains the base SLE Micro image to modify, along with any other custom files that
+   will be included in the built image
+
+#### Image Configuration File
+
+The image configuration file is a YAML document describing a single image to build. The file is specified using
+the `-config-file` argument. Only a single image may be built at a time, however the same image configuration
+directory may be used to build multiple images by creating multiple configuration files.
+
+The following can be used as the minimum configuration required to create an image:
+```yaml
+apiVersion: 1.0
+image:
+  imageType: iso
+  baseImage: SLE-Micro.x86_64-5.5.0-Default-SelfInstall-GM.install.iso
+  outputImageName: eib-image.iso
+```
+
+* `imageType` - Must be either `iso` or `raw`.
+* `baseImage` - Indicates the name of the image file used as the base for the built image. This file must be located
+  under the `images` directory of the image configuration directory (see below for more information). This image will
+  **not** directly be modified by EIB; a new image will be created each time EIB is run.
+* `outputImageName` - Indicates the name of the image that EIB will build. This may only be a filename; the image will
+  be written to the root of the image configuration directory.
+
+#### Image Configuration Directory
+
+The image configuration directory contains all the files necessary for EIB to build an image. As the project matures,
+the structure of this directory will be better fleshed out. For now, the required structure is described below:
+
 ```shell
 .
-├── eib-config.yaml
+├── eib-config-iso.yaml
+├── eib-config-raw.yaml
 └── images
     └── SLE-Micro.x86_64-5.5.0-Default-SelfInstall-GM.install.iso
+    └── SLE-Micro.x86_64-5.5.0-Default-GM.raw
 ```
 
-That directory must be attached to the container at runtime. The following command attaches the directory and runs
-EIB against the volume (replace `$IMAGE_DIR` with your local configuration directory):
+* `eib-config.yaml` - All image configuration files should be in the root of the image configuration directory.
```

Comment:
```suggestion
* `eib-config-iso.yaml`, `eib-config-raw.yaml` - All image configuration files should be in the root of the image configuration directory.
```
The specified file no longer exists in the new dir structure.

Reply:
Good eye, ty.